### PR TITLE
#38: Table comments need escaping

### LINKED ISSUE #28: Refactor for better organization and class names
## Background

The little chaotic code from hackathon needs better organization and naming

## Feature

Rename and move classes around

### LINKED ISSUE #34: Support tables description
## Background
Tables can have comment explaining their purpose.

## Feature
Both yaml and pg query already support description field of table, but it's not applied.
And the needed sqls.

### LINKED ISSUE #38: Table comments need escaping
## Background
Table and column comments need escaping. If not for the danger of SQL injection (whoever write the source code can put anything there), then at least to allow `'` and `\` characters in the comments

## Feature
Table and column comments are escaped.

### PR DESCRIPTION
**Pull request by benedeki**
* Implemented table comment (description)
* Handling ' character in comments
* Unified setting and clearing of column comment (description)

Closes #38 
Closes #34
Depends on #42

### CODE REVIEW

**Comment by benedeki** [comment]
This item depends on:
* AbsaOSS/ultet#42
By **[Dependent Issues](https://github.com/z0al/dependent-issues)**.

**Comment by benedeki** [comment]
This item depends on:
* ~~AbsaOSS/ultet#42~~
By **[Dependent Issues](https://github.com/z0al/dependent-issues)**.

File: src/main/scala/za/co/absa/ultet/model/table/TableDef.scala [hunk 1]
```diff
@@ -141,10 +141,10 @@ object TableDef {
...
 
     private def generateAlterForDescriptionChange(thisCol: TableColumn, otherCol: TableColumn): Seq[TableAlteration] = {
-      (thisCol.description, otherCol.description) match {
-        case (t, o) if t == o => Seq.empty // no change
-        case (Some(_), None) => Seq(TableColumnCommentDrop(tableIdentifier, thisCol.columnName))
-        case (_, Some(o)) => Seq(TableColumnCommentSet(tableIdentifier, otherCol.columnName, o)) // both add/set
+      if (thisCol.description != otherCol.description) {
+        Seq(TableColumnDescriptionSet(tableIdentifier, otherCol.columnName, otherCol.description)) // add/un-/set
+      } else {
+        Seq.empty // no change
```
Comment (lsulak) [inline src/main/scala/za/co/absa/ultet/model/table/TableDef.scala:147]: btw on line 154, in func `generateAlterForDefaultChange`, `case (Some(t), None)` -> `case (_, None)`

**Review by lsulak — APPROVED** [review]
LGTM, just read the code but I like that you wrote some tests as well that document the code nicely!

**Review by lsulak — APPROVED** [review]

**Pull request merged**